service/src/test/java/uk/gov/hmcts/reform/fpl/enums/ChildGenderTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.ChildGender.BOY;
+import static uk.gov.hmcts.reform.fpl.enums.ChildGender.GIRL;
+import static uk.gov.hmcts.reform.fpl.enums.ChildGender.OTHER;
+
+public class ChildGenderTest {
+
+    @Test
+    public void shouldReturnBoy() {
+        assertThat(ChildGender.fromLabel(BOY.getLabel())).isEqualTo(BOY);
```

Comment:
Isn't this assertion the same as the last one?
Also thoughts on making this and the others parameterised?

Reply:
'accidentially' it is same, but conceptually it is different. Wanted to be sure that enum->label->enum deserialization works
changed to parametrised